web/static/js/components/user_list_item.jsx

```diff
@@ -8,18 +10,35 @@ import STAGES from "../configs/stages"
 
 const { VOTING } = STAGES
 
-export const UserListItem = ({ user, votes, stage }) => {
+export const UserListItem = ({ user, votes, stage, currentUser, actions }) => {
   const imgSrc = user.picture.replace("sz=50", "sz=200")
   const votesByUser = votes.filter(vote => vote.user_id === user.id).length
   const allVotesIn = votesByUser >= VOTE_LIMIT
+  const renderButton = !user.is_facilitator && currentUser.is_facilitator
 
   let identifier = user.given_name
   if (user.is_facilitator) {
     identifier += " (Facilitator)"
   }
 
+  function passFacilitatorship(user) {
+    if (window.confirm(`Do you want ${identifier} to be Facilitator?`)) {
+      actions.updateRetroAsync({ facilitator_id: user.id })
+    }
+  }
+
   return (
     <li className={`item ${styles.wrapper}`}>
+      {renderButton
+      && (
+        <button
+          type="button"
+          className={`${styles.facilitator}`}
+          onClick={() => passFacilitatorship(user, currentUser)}
+        >
+          {currentUser !== user && <i className="magic icon" />}
```

Comment:
i think when we extract this, we'll want to do a comparison of the `currentUser.id` and `user.id`, as that will protect us if they are two separate objects that are identical in contents, which is possible, especially in redux apps due to the preferring transformations to mutations

Reply:
hadn't considered that 🤔 good looks